packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEdit.tsx

```diff
@@ -235,29 +230,27 @@ export const InboundLineEdit: FC<InboundLineEditProps> = ({
                   borderRadius: '20px',
                 }}
               >
-                <Box
-                  sx={{
-                    width: 400,
-                    height: isMediumScreen ? 300 : 400,
-                    backgroundColor: theme =>
-                      alpha(theme.palette['background']['menu'], 0.4),
-                    position: 'absolute',
-                    borderRadius: '20px',
-                  }}
-                />
-                <StyledTabPanel value={Tabs.Batch}>
-                  <BatchTable
+                <InboundLineEditPanel
```

Comment:
I wonder if a Slide animation would work for the tab switching - to reinforce the idea that the first couple of columns are static and the other columns are swapped in and out?

Reply:
Yes I wondered the same and added it - but then removed it 😁 

I kind of just felt like it was a little too distracting and busy and felt like when I was switching tab the time for the animation was annoying. Maybe I'll add it back and throw it up as a draft to see what you think.

One thing that is super cool is the animation on add batch, though!!